{"uuid": "c75d78fa-6222-405b-9f94-4a835742a9e3", "children": ["42fcbef9-9cef-44c6-a10c-4b7fc4405cc4"], "befores": [{"name": "cart_page_with_items", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'https://www.saucedemo.com/' == 'https://www.saucedemo.com/inventory.html'\n  \n  - https://www.saucedemo.com/inventory.html\n  ?                           --------------\n  + https://www.saucedemo.com/\n", "trace": "  File \"C:\\Users\\dhamyan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\dhamyan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\dhamyan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\dhamyan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\dhamyan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\dhamyan\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\dhamyan\\PycharmProjects\\PythonProject\\PythonProject\\Automated-testing-project\\test\\test_cart.py\", line 27, in cart_page_with_items\n    assert login_page.driver.current_url == \"https://www.saucedemo.com/inventory.html\"\n"}, "start": 1759173798516, "stop": 1759173798728}], "start": 1759173798516, "stop": 1759173798735}